client/sites-dashboard/components/link-in-bio-banner/BannerType.ts

```diff
@@ -0,0 +1,6 @@
+export enum BannerType {
```

Comment:
We try to avoid using TypeScript enums because they behave a little strange p7H4VZ-3sK-p2.

In this case I'd use a union type whenever I needed to reference all the banner types together.
```typescript
export type BannerType = 'none' | 'tile' | 'double-tile' | 'row';
```

Personally I don't think it's necessary, but if you prefer having named constants you'd just make  a constant for each one i.e. `export const BANNER_TYPE_NONE = 'none'`

Reply:
I will change that then. Thanks.